formatTest/typeCheckedTests/input/jsx.re

```diff
@@ -59,6 +59,15 @@ module Namespace = {
     };
 };
 
+module Optional = {
+    let createElement ::optional children => {
```

Comment:
From what I see it works correctly, as this is a required option value.

I did add the extra tests as there is a slight issue with partial application in the JSX tests.